internal/server/server.go

```diff
@@ -258,3 +268,33 @@ func (ss *StreamerServer) handleStartStreams(w http.ResponseWriter, r *http.Requ
 	}
 	w.Write(res)
 }
+
+// Set the orchestators discoverable by the broadcaster using the orchWebhook
+func (ss *StreamerServer) handleOrchestrators(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+
+	ss.lock.Lock()
```

Comment:
Can `RLock()`/`RUnlock()` be used here and below since we're just reading and not writing?

Reply:
fixed in cf8b1fa